### DISCUSSION TITLE
Is libssh or libssh2 the default library for `ssh` command?

### DISCUSSION THREAD

**Discussion by l2dy** [blinksh/blink#1919] [Q&A] [answered]
README still attributes libssh2. Is libssh recently added?

**Comment by carloscabanero** [comment, marked as answer]
Libssh is for ssh.
Libssh2 is on ssh2 (deprecated since long time ago)

**Reply by l2dy** [reply]
@carloscabanero Has the license situation of libssh changed since https://github.com/blinksh/mosh/issues/3#issuecomment-276991097?

**Reply by carloscabanero** [reply]
Yeah, so that is a really old message, and back in the day I did not know better. I consulted lawyers and now there is also previous precedent on this. Closed source companies had been forced to release their object code so that people could recompile with a version of a library that you can tweak yourself, etc... So I think we are good with LGPL.

Now, I am less clear if we could use GPL and if the GPL still clashes with the AppStore. Other Open Source apps have bundled GPL code from others though, and they think that because they give a "clause" themselves like Mosh and we do, that magically applies to the other code as well. I'm not so sure about that, but Apple is fine accepting those apps. I'm not going to risk it.
